docs/999-big-list-of-options.md

```diff
@@ -1096,6 +1096,13 @@ Default: `true`
 
 Whether to clear the screen when a rebuild is triggered.
 
+#### watch.skipWrite
+Type: `boolean`<br>
+<!-- CLI: `--watch.skipWrite`<br> -->
```

Comment:
This CLI option _will_ work, but I figure it's better not to list it (similar to `watch.chokidar`) since its application makes zero sense via the CLI layer.